integration/cmd/search-adhoc-request/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+
+	"github.com/Telenav/osrm-backend/integration/pkg/api/search/coordinate"
+	"github.com/Telenav/osrm-backend/integration/pkg/api/search/nearbychargestation"
+)
+
+func main() {
+	flag.Parse()
+
+	if len(flags.entityEndpoint) == 0 {
+		fmt.Println("[ERROR] -entity should not be empty!")
+		return
+	}
+
+	if len(flags.apiKey) == 0 {
+		fmt.Println("[ERROR] -apikey should not be empty.")
+		return
+	}
+	if len(flags.apiSignature) == 0 {
+		fmt.Println("[ERROR] -apisignature should not be empty.")
+		return
+	}
+	req := nearbychargestation.NewRequest()
+	req.APIKey = flags.apiKey
+	req.APISignature = flags.apiSignature
+	req.Location = coordinate.Coordinate{Lat: 37.78509, Lon: -122.41988}
+
+	clt := http.Client{Timeout: 60000000000}
```

Comment:
There's a `backend.Timeout()` to unify this value by command line flag, please have a try.

Reply:
Ok, that one is better.